openpype/hosts/blender/plugins/publish/validate_setdress_cam_instance.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+import bpy
+import string
+
+import pyblish.api
+
+import openpype.hosts.blender.api.action
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateSetdressCamInstance(pyblish.api.Validator):
+    """Validate that cam and review instances points on the right collection."""
+
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    families = ["setdress"]
+    label = "Validate SetDress Cam Review Instances"
```

Comment:
This is incorrect. `setdress` is a family, the step is Environment. Here you deal with Camera and review instances, explain how.